CRM/Upgrade/Incremental/sql/5.5.alpha1.mysql.tpl

```diff
@@ -1 +1,7 @@
 {* file to handle db changes in 5.5.alpha1 during upgrade *}
+
+{* Fix is_reserved flag on civicrm_option_group table *}
+UPDATE civicrm_option_group AS cog INNER JOIN civicrm_custom_field AS ccf
+ON cog.id = ccf.option_group_id
+SET cog.is_reserved = 0 WHERE cog.is_active = 1 AND ccf.is_active = 1;
+UPDATE civicrm_option_group SET is_reserved = 1 WHERE name='environment';
```

Comment:
The upgrade code does not affect new installs, so the `environment` group needs to be set to reserved in the install sql as well.

Reply:
@colemanw I've updated the install sql as well now.